e2e-tests/gef/cypress/e2e/amendments/portal-amendments-disabled/make-a-change-button.spec.js

```diff
@@ -0,0 +1,66 @@
+import { FACILITY_TYPE } from '@ukef/dtfs2-common';
+import relative from '../../relativeURL';
+import CONSTANTS from '../../../fixtures/constants';
+import { MOCK_APPLICATION_AIN } from '../../../fixtures/mocks/mock-deals';
+import { BANK1_MAKER1, BANK1_CHECKER1_WITH_MOCK_ID } from '../../../../../e2e-fixtures/portal-users.fixture';
+import { multipleMockGefFacilities } from '../../../../../e2e-fixtures/mock-gef-facilities';
+import applicationPreview from '../../pages/application-preview';
+
+const { unissuedCashFacility, unissuedContingentFacility, anIssuedContingentFacility, issuedCashFacility } = multipleMockGefFacilities();
+
+let dealId;
+let token;
+
+let issuedCashFacilityId;
+let issuedContingentFacilityId;
+let unissuedCashFacilityId;
+let unissuedContingentFacilityId;
+
+context('Make a change button - FF_PORTAL_FACILITY_AMENDMENTS_ENABLED feature flag disabled', () => {
+  before(() => {
+    cy.apiLogin(BANK1_MAKER1)
+      .then((t) => {
+        token = t;
+      })
+      .then(() => {
+        // creates Acknowledged AIN application and inserts facilities
+        cy.apiCreateApplication(BANK1_MAKER1, token).then(({ body }) => {
+          dealId = body._id;
+          cy.apiUpdateApplication(dealId, token, MOCK_APPLICATION_AIN);
+          cy.submitDealAfterUkefIds(dealId, 'GEF', BANK1_CHECKER1_WITH_MOCK_ID);
+          cy.apiUpdateApplication(dealId, token, MOCK_APPLICATION_AIN).then(() => {
+            cy.apiCreateFacility(dealId, FACILITY_TYPE.CASH, token).then((facility) => {
+              issuedCashFacilityId = facility.body.details._id;
+              cy.apiUpdateFacility(facility.body.details._id, token, issuedCashFacility);
+            });
+            cy.apiCreateFacility(dealId, FACILITY_TYPE.CASH, token).then((facility) => {
+              unissuedCashFacilityId = facility.body.details._id;
+              cy.apiUpdateFacility(facility.body.details._id, token, unissuedCashFacility);
+            });
+            cy.apiCreateFacility(dealId, FACILITY_TYPE.CONTINGENT, token).then((facility) => {
+              unissuedContingentFacilityId = facility.body.details._id;
+              cy.apiUpdateFacility(facility.body.details._id, token, unissuedContingentFacility);
+            });
+            cy.apiCreateFacility(dealId, CONSTANTS.FACILITY_TYPE.CONTINGENT, token).then((facility) => {
```

Comment:
Can we add a new line after each `cy.apiCreateFacility` please

Reply:
Good idea, added